src/components/navigation/navItem/index.test.js

```diff
@@ -1,28 +1,17 @@
 import React from 'react'
 import { shallow, mount } from 'enzyme'
 import toJSON from 'enzyme-to-json'
-import 'jest-styled-components'
 import NavItem from '../navItem'
 import { mockData } from './__mocks__'
 
 describe('Desktop version <NavItem/>', () => {
   beforeAll(() => {
-    global.___loader = {
-      enqueue: jest.fn(),
-    }
-
     global.window.matchMedia = jest.fn(() => {
       return {
         matches: true,
-        addListener: jest.fn(),
-        removeListener: jest.fn(),
       }
     })
   })
```

Comment:
Following the test suite running, you'll need to reset the `global.window.matchMedia` mock to what it was previously. This could (for example) be done by capturing the original functionality and re-applying it in the `afterAll`.

Reply:
Fixed.